episodes/quality-control-and-manual-segmentation.md

```diff
@@ -0,0 +1,547 @@
+values to the right. In an ideal world the image could have been acquired with a 
+longer exposure time / higher laser power etc to collect more light. This slight 
+underexposure isn't a big issue here though, as we can still clearly separate 
+the nuclei from the background which is our main aim.
+
+Most importantly, we see no evidence for 'clipping', which means we are avoiding 
+any irretrievable loss of information this would cause. Recall that clipping 
+occurs when pixels are recording light above the maximum limit for the image. 
+Therefore, many values are 'clipped' to the maximum value resulting in 
+information loss.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## What is segmentation?
+
+In order to count the number of cells, we must 'segment' the nuclei in this 
+image. Segmentation is the process of figuring out what each pixel in an image 
+represents e.g. is that pixel part of a nucleus or not? Segmentation comes in 
+two main types - 'semantic segmentation' and 'instance segmentation'. 
+
+Let's take a quick look at a rough semantic segmentation. Open Napari's console 
+by pressing the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/console.svg
+){alt="A screenshot of Napari's console button" height='30px'} button, then copy 
+and paste the code below. Don't worry about the details of what's happening in 
+the code - we'll look at some of these concepts like gaussian blur and otsu 
+thresholding in later episodes!
+
+```python
+from skimage.filters import threshold_otsu, gaussian
+
+image = viewer.layers["nuclei"].data
+
+blurred = gaussian(image, sigma=3)
+threshold = threshold_otsu(blurred)
+
+semantic_seg = blurred > threshold
+viewer.add_labels(semantic_seg)
+```
+
+![](fig/semantic-seg-napari.png){alt="A screenshot of a rough semantic 
+segmentation of nuclei in Napari"}
+
+You should see an image appear that highlights the nuclei in brown. This is an 
+example of a 'semantic segmentation'. In a semantic segmentation, pixels are 
+grouped into different categories (also known as 'classes') - for example, 
+nuclei vs background. Importantly, it doesn't recognise which pixels belong to 
+different objects of the same category - for example, here we don't know which 
+pixels belong to individual, separate nuclei. This is the role of 'instance 
+segmentation' that we'll look at next.
+
+Copy and paste the code below into Napari's console. Again, don't worry about 
+the details of what's happening here - we'll look at some of these concepts in 
+later episodes.
+
+```python
+from skimage.morphology import binary_erosion, ball
+from skimage.segmentation import expand_labels
+from skimage.measure import label
+
+eroded = binary_erosion(semantic_seg, footprint=ball(10))
+instance_seg = label(eroded)
+instance_seg = expand_labels(instance_seg, distance=10)
+
+viewer.add_labels(instance_seg)
+```
```

Comment:
Should each code snippet be stand alone? This will not work unless we have semantic_seg from the previous snippet. We could add 
`semantic_seg = viewer.layers["semantic_seg"].data`
Maybe it's not worth it as it won't work unless that layer is in the image anyway.